middlewares/team.middleware.js

```diff
@@ -165,6 +165,35 @@ async function findById(req, res, next) {
     return next();
 }
 
+/**
+ * @async
+ * @function deleteHackerFromTeam
+ * @param {{user: {id: ObjectId}} req 
+ * @param {*} res 
+ * @return {JSON} Success or error status
+ * @description Removes the hacker from the team under teamId. If hacker is not part of a team, it does nothing.
+ */
+async function deleteHackerFromTeam(req, res, next) {
```

Comment:
Nitpick, but what do you think about changing the name to 'deleteUserFromTeam', as we are utilizing req.user.id, which is not immediately obvious from the function name. Alternatively, I suggest using req.body.hackerId to retrieve the hacker, and first use some middleware that gets the hackerId from userId. It may or may not exist already.

Reply:
Valid point, I'll change the name to `deleteUserFromTeam`